### PR TITLE
Fix formatting

### PR DESCRIPTION
**Pull request by leshi**

### CODE REVIEW

File: u2f-gae-demo/src/com/google/u2f/gaedemo/storage/TokenStorageData.java [hunk 1]
```diff
@@ -119,8 +113,7 @@ public boolean equals(Object obj) {
     TokenStorageData that = (TokenStorageData) obj;
     return (this.enrollmentTime == that.enrollmentTime)
         && SecurityKeyData.containSameTransports(this.transports, that.transports)
-        && (this.counter == that.counter)
-        && Arrays.equals(this.keyHandle, that.keyHandle)
+        && (this.counter == that.counter) && Arrays.equals(this.keyHandle, that.keyHandle)
```
Comment (juanlang) [inline u2f-gae-demo/src/com/google/u2f/gaedemo/storage/TokenStorageData.java]: Not positive this is better. It's true that these fit on one line. On the other hand, leaving it as it was allows you to scan each predicate one line at a time.

File: u2f-ref-code/java/src/com/google/u2f/server/impl/U2FServerReferenceImpl.java [hunk 2]
```diff
@@ -93,8 +94,8 @@ public RegistrationRequest getRegistrationRequest(String accountName, String app
   }
 
   @Override
-  public SecurityKeyData processRegistrationResponse(RegistrationResponse registrationResponse,
-      long currentTimeInMillis) throws U2FException {
+  public SecurityKeyData processRegistrationResponse(
+      RegistrationResponse registrationResponse, long currentTimeInMillis) throws U2FException {
```
Comment (juanlang) [inline u2f-ref-code/java/src/com/google/u2f/server/impl/U2FServerReferenceImpl.java]: throws clause on a separate line preferred, I think?

File: u2f-ref-code/java/src/com/google/u2f/server/impl/U2FServerReferenceImpl.java [hunk 3]
```diff
@@ -266,24 +269,24 @@ public SecurityKeyData processSignResponse(SignResponse signResponse) throws U2F
...
 
     Log.info("<< processSignResponse");
     return securityKeyData;
   }
 
-  private void verifyBrowserData(JsonElement browserDataAsElement,
-      String messageType, EnrollSessionData sessionData) throws U2FException {
-
+  private void verifyBrowserData(JsonElement browserDataAsElement, String messageType,
+      EnrollSessionData sessionData) throws U2FException {
```
Comment (juanlang) [inline u2f-ref-code/java/src/com/google/u2f/server/impl/U2FServerReferenceImpl.java]: ditto

File: u2f-gae-demo/src/com/google/u2f/gaedemo/storage/TokenStorageData.java (hunk 1 above)
Comment (leshi) [inline u2f-gae-demo/src/com/google/u2f/gaedemo/storage/TokenStorageData.java]: This is the official eclipse formatting style.  I wanted to go with it so we don't have to format only parts of the file automatically.  But won't push.  Can revert this. if you'd like.

File: u2f-ref-code/java/src/com/google/u2f/server/impl/U2FServerReferenceImpl.java (hunk 2 above)
Comment (leshi) [inline u2f-ref-code/java/src/com/google/u2f/server/impl/U2FServerReferenceImpl.java]: I looked and couldn't find anything like that in the style guide.  So I went with the default eclipse google style plugin decision.

File: u2f-gae-demo/src/com/google/u2f/gaedemo/storage/TokenStorageData.java (hunk 1 above)
Comment (juanlang) [inline u2f-gae-demo/src/com/google/u2f/gaedemo/storage/TokenStorageData.java]: The auto-formatter is bad, and it should feel bad.

Meh. OK.

**Comment by juanlang** [comment]
Autoformatting FTW?

**Pull request merged**